headWrongEnd/solution.js

```diff
@@ -0,0 +1,4 @@
+function fixTheMeerkat(arr) {
+    var meerkat = arr;
```

Comment:
You don't have to assign arr to another variable, you can just return arr.reverse();